third_party/3/pyspark/sql/column.pyi

```diff
@@ -81,6 +81,7 @@ class Column:
     def alias(self, *alias: str, **kwargs: Any) -> Column: ...
     def name(self, *alias: str) -> Column: ...
     def cast(self, dataType: Union[DataType, str]) -> Column: ...
+    def contains(self, other: Union[DataType, str]) -> Column: ...
```

Comment:
I'd prefer to keep this consistent with [upstream](https://github.com/apache/spark/blob/94bd480761b9f0cf8dc5b87421a4f63480d8e3a9/python/pyspark/sql/column.pyi#L118) and have `Any` here (see https://github.com/apache/spark/pull/31823#pullrequestreview-611607868)